### DISCUSSION TITLE
Bulk selection added/counted twice for every checkbox

### DISCUSSION THREAD

**Discussion by AndMe2** [rappasoft/laravel-livewire-tables#1457] [Q&A]
Hi!
I have a table which I have a bulk action on, it works. But for every checkbox I check it gets added twice, and the ID is represented twice. Also works in reverse, so when unchecked both get removed.
I have several tables on the same page, but they have different ID:s and tableNames, this is the only one with bulk actions activated.
I have upgraded and also reinstalled the whole package in composer. It is now version 2.15.0
Theme is bootstrap-5.
Could It be something with tables having ->setPrimaryKey('id')?
Any Ideas what it can be?

This is the config for the table:
    public function configure(): void
    {
        $this->setPrimaryKey('id')
        ->setEagerLoadAllRelationsStatus(true)
        ->setEagerLoadAllRelationsEnabled()
        ->setBulkActions([
            'printSelected' => Lang::get('lang.print_selected'),
        ])
        ->setTableWrapperAttributes([
            'class' => 'dynheight',
          ])
          ->setTheadAttributes([
            'class' => 'datatable__thead',
          ])
          ->setPerPageAccepted([25, 50, 100, -1])
          ->setPerPage(25)
          ->setTdAttributes(function() {
            return [
              'default' => true,
              'class' => 'py-1',
            ];
        });
    }
    
    
![scrn_rappasoftDatatables](https://github.com/rappasoft/laravel-livewire-tables/assets/42405689/fc1ebf57-24ed-46f5-a71d-772da4f7933c)

**Comment by lrljoe** [comment]
Please can you raise an Issue for this, so that we can track it appropriately :)

Please include the full table component if possible as that helps diagnose the issue!
